src/v1.rs

```diff
@@ -189,37 +189,28 @@ impl Party {
         }
     }
 
-    /// Sign `msg` with this party's share of the group private key, using the set of `sigers` and corresponding `nonces` with a precomputed `aggregate_nonce`
+    /// Sign `msg` with this party's share of the group private key, using the set of `sigers` and corresponding `nonces` with a precomputed `aggregate_nonce` and an optional tweak to the public key
```

Comment:
Don't think this comment is accurate anymore? Doesn't seem like the tweak is optional?

Reply:
Fixed